packages/lexical-editor-cms-actions/src/components/LexicalCmsEditor.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import {
+    RichTextEditor,
+    RichTextEditorProps
+} from "@webiny/lexical-editor/components/Editor/RichTextEditor";
+import { WebinyListPlugin } from "@webiny/lexical-editor/plugins/WebinyListPLugin/WebinyListPlugin";
+import {
+    ClickableLinkPlugin,
+    CodeHighlightPlugin,
+    FloatingLinkEditorPlugin,
+    LinkPlugin
+} from "@webiny/lexical-editor";
+import { usePageElements } from "@webiny/app-page-builder-elements";
+import { RichTextStaticToolbar } from "~/components/RichTextStaticToolbar";
+
+interface RichTextContentEditorProps extends Omit<RichTextEditorProps, "theme"> {
+    tag?: "p";
+    toolbarActionPlugins?: { type: string; plugin: Record<string, any> }[];
+    theme?: Record<string, any>;
+}
+
+export const LexicalCmsEditor: React.FC<RichTextContentEditorProps> = ({
+    placeholder,
+    toolbarActionPlugins,
+    tag,
+    ...rest
+}) => {
+    const { theme } = usePageElements();
```

Comment:
I see `usePageElements` here. Not sure whether what we discussed today with Bruno/Monika was something that we'll do later, or we want to do it now?

If you ask me, it's now.